rtc_data_service/src/attestation/service.rs

```diff
@@ -0,0 +1,22 @@
+use actix::Addr;
+use actix_web::{error::ErrorInternalServerError, get, web, HttpRequest};
+
+use crate::auth_enclave_actor::AuthEnclaveActor;
+use crate::merge_error::*;
+
+use super::AttestationMessage;
+
+#[get("auth/attest")]
```

Comment:
We probably want a leading `/` here, for consistency?

(I can't find any clear documentation on whether the leading `/` is actually required to anchor the path or not, but all the Actix Web examples include it.)